README.md

```diff
@@ -7,6 +7,10 @@ JavaFX CSS theme collection plus additional controls. It's inspired by [FlatLaf]
 ![primer_light](./.screenshots/primer_light.png)
 ![primer_dark](./.screenshots/primer_dark.png)
 
+## Try it out
+
+Grab a **[self-updating download of the Sampler app](https://downloads.hydraulic.dev/atlantafx/sampler)** for Windows, macOS and Linux, packaged with [Conveyor](https://www.hydraulic.software).
```

Comment:
Shouldn't this link be

https://downloads.hydraulic.dev/atlantafx/sampler/download.html

the link without the /download.html currently 404s.

Reply:
Oof, yes, you're right. I just pushed a fixed commit - thanks!